config/crd/v1/bases/velero.io_restores.yaml

```diff
@@ -186,6 +186,12 @@ spec:
                                     - Continue
                                     - Fail
                                     type: string
+                                  waitForReady:
```

Comment:
Looks like this is not relevant to the issue, what is the reason to make this change?
And as the principle to change a CRD, the yaml file is auto-generated, and should not be changed manually. We should change the `/pkg/api/velero/vx/*_types.go` only.

Reply:
CRDs were out of date from a prior PR.

Reply:
If someone would PR a `make update` only change and merge that I can rebase to make this PR this issue specific.
Another similar case
https://github.com/vmware-tanzu/velero/pull/7003/files#diff-49ea52a49c8a35bb42fdef90a38cc231783e10d8072be60b31cc884aa066cb74

Just didn't want to leave CI failing.

Reply:
Got it, thanks.